### PR TITLE
support synthetic for object detection

### PR DESCRIPTION
**Pull request by mdrozdowski1996**

### CODE REVIEW

File: tftrt/examples/object_detection/object_detection.py [hunk 1]
```diff
@@ -539,88 +544,102 @@ def benchmark_model(frozen_graph,
...
-                    image = _read_image(image_path, image_shape)           
-                    batch_images.append(image)
-
-                # run once outside of timing to initialize
-                if image_idx == 0:
+            for image_idx in tqdm.tqdm(range(0, num_images, batch_size)):
+
+                if use_synthetic:
+                    if image_shape is not None:
+                        batch_images = np.random.randint(256, size=(batch_size, 600, 600, 3))
```
Comment (mdrozdowski1996) [inline tftrt/examples/object_detection/object_detection.py]: Should I define standard image size (600, 600, 3) before use it?

Comment (trevor-m) [reply tftrt/examples/object_detection/object_detection.py]: change the default in benchmark_model to [600, 600, 3]

File: tftrt/examples/object_detection/object_detection.py [hunk 2]
```diff
@@ -40,6 +40,9 @@
 from object_detection.protos import pipeline_pb2, image_resizer_pb2
 from object_detection import exporter
 
+from pycocotools.coco import COCO
```
Comment (pooyadavoodi) [inline tftrt/examples/object_detection/object_detection.py]: Why is capital letters used for coco?

Comment (pooyadavoodi) [reply tftrt/examples/object_detection/object_detection.py]: Never mind. It comes from pycocotools.

**Pull request merged**